server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAnalysesList.js

```diff
@@ -138,7 +140,7 @@ export default class ProjectActivityAnalysesList extends React.PureComponent {
     }
   };
 
-  updateSelectedDate = (date: Date) => this.props.updateQuery({ selectedDate: date });
+  updateSelectedDate = (date /*: Date */) => this.props.updateQuery({ selectedDate: date });
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)